content/en/docs/tasks/security/authentication/mtls-migration/index.md

```diff
@@ -145,11 +210,88 @@ If you can't migrate all your services to Istio (injecting Envoy sidecar), you h
 However, when configured with `PERMISSIVE` mode, no authentication or authorization checks will be performed for plaintext traffic by default.
 We recommend you use [Istio Authorization](/docs/tasks/security/authorization/authz-http/) to configure different paths with different authorization policies.
 
-## Cleanup
+### Cleanup
 
-Remove all resources.
+To remove all temperary resources created in this task:
 
 {{< text bash >}}
 $ kubectl delete ns foo bar legacy
 Namespaces foo bar legacy deleted.
 {{< /text >}}
+
+## Globally enable mutual TLS for the cluster
+
+This section describes how to apply the cluster-wide DestinationRule and MeshPolicy to enforce
+mutual TLS for a cluster. For more details, please read the
+[Authentication policy](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls)
+task.
+
+{{< warning >}}
+If you have special TLS configurations for your services or you have
+services without Envoy sidecars, we recommend you to enable mutual TLS service by service.
```

Comment:
It's better to add a link to the above section to help the user to quickly navigate to the other option and avoid confusion.

Reply:
Done.